app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryOwnedBy.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default ExpandedStoryOwnedBy = (props) => {
+  const { users, story, onEdit } = props;
+
+  return (
+    <div className="Story__section">
+      <div className="Story__section-title">
```

Comment:
Using `div`'s for titles is not semantic, what about using a heading here instead?

Reply:
I think it's a good idea but since we're gonna abstract a `StorySection` component in a follow-up PR we can fix it there for every place we use this same portion of code, what do you think? @kostadriano add that to our refactor :)